app/Models/Post.php

```diff
@@ -217,13 +218,18 @@ public function toBlinkPayload()
         // Blink expects quantity to be a number.
         $quantity = $this->quantity === null ? 0 : $this->quantity;
 
+        // Fetch Campaign Website information via GraphQL.
+        $campaignWebsite = app(GraphQL::class)->getCampaignWebsiteByCampaignId($this->campaign_id);
+
         return [
             'id' => $this->id,
             'signup_id' => $this->signup_id,
             'quantity' => $quantity,
             'why_participated' => $this->signup->why_participated,
             'campaign_id' => (string) $this->campaign_id,
             'campaign_run_id' => (string) $this->signup->campaign_run_id,
+            'campaign_title' => array_get($campaignWebsite, 'title'),
+            'campaign_slug' => array_get($campaignWebsite, 'slug'),
```

Comment:
Since these are selected in the query we'll always have at least `null` for these values right?

Reply:
Based on my testing, if there's no matching entry for the provided `campaignId`, it'll just send back `null` as a top-level value without the fields themselves - even if they're specified in the query.

Buuut it seems we could get away with just grabbing the values without the `array_get` and it still wouldn't fail which is surprising to me! I had no idea that `null['index']` just returns `null`, or am I tripping?

![image](https://user-images.githubusercontent.com/12417657/60552607-74b21e80-9cfd-11e9-99be-1d228ca3efaf.png)

So we could just have 
```php
...
'campaign_title' => $campaignWebsite['title'],
'campaign_slug' => $campaignWebsite['slug'],
...
```
And would we prefer that to the `array_get`s?

Reply:
Yeah I think that style is better, nothing is nested here and you're not using any dot notation which I believe is the upside of [`array_get`](https://laravel.com/docs/5.2/helpers#method-array-get)! Most important though is that it won't error out no matter what it gets back from GraphQL which it sounds like it won't, my tests showed the same results as yours!

Reply:
Makes sense! updated in https://github.com/DoSomething/rogue/pull/890/commits/7849807c00e1e4a4330e541cd6862378b7ba1a3b